tasks/Functions-client/request.js

```diff
@@ -230,3 +245,41 @@ task("functions-request", "Initiates a request from an Functions client contract
       console.log(`Waiting for fulfillment...\n`)
     })
   })
+
+
+const verifyOffchainSecrets = async (secretsURLs, nodeAddresses) => {
```

Comment:
Reuse code instead of copy-pasting.

Reply:
The `verifyOffchainSecrets` command is not used anywhere else except `request.js`.

It is placed in a separate function to avoid further confusing the existing logic in `request.js`.

Reply:
It is used in buildRequestJSON.js - are you planning to share that one?

Reply:
No.  `verifyOffchainSecrets` checks the URLs where `buildRequestJSON.js` builds the JSON file which must be uploaded.  I will rename the function for clarity.

Reply:
My point is that there's literally the same piece of code in two PRs so why not put it in a common place to reuse it?

Reply:
I see where I missed the repeated use.  It was because I had already moved buildRequestJSON into a different PR.

Reply:
Since these 2 PRs have been split, this must be addressed after the PRs are merged (see my comments in the `functions-build-request` PR)

Reply:
Let's merge PR #59 and then rebase this one and address it here.